daemon/daemon.go

```diff
@@ -369,6 +369,9 @@ configRetry:
 	} else {
 		// Use the syncer locally.
 		syncer = felixsyncer.New(backendClient, datastoreConfig.Spec, syncerToValidator)
+
+		log.Info("using resource updates where applicable")
+		configParams.SetUseResourceUpdates(true)
```

Comment:
I think same here - in general we don't need to use Setters / Getters since configParams isn't a public API.

Reply:
Is the plan to do the Typha support for this in a follow-on PR?

Reply:
Yes, typha support will have to come after this

Reply:
I realised I didn't comment on your first comment, but this is a bit different because we don't know if we want to use the resource updates until this point

Reply:
@Brian-McM have you started a PR for the Typha changes?

Reply:
I haven't, I've made a task for it and Raphael has volunteered to help with it